### PR TITLE
Jtac talk over srs

### PR DESCRIPTION
**Pull request by davidp57**
I've implemented the JTAC over SRS feature I told you about earlier.
Also, I've edited the `readme.md` file with a concise documentation.

Tell me if you want me to do something else.

### CODE REVIEW

**Comment by davidp57** [comment]
Hi @ciribob ; as requested, I rebased.
Nothing to report except that I changed the URL to our team's Github (you used my personal one)

File: DCS-SimpleTextToSpeech.lua [hunk 1]
```diff
@@ -0,0 +1,215 @@
+--[[
+
+DCS-SimpleTextToSpeech
+Version 0.4
+Compatible with SRS version 1.9.6.0 +
```
Comment (ciribob) [inline DCS-SimpleTextToSpeech.lua:5]: Can you please delete this file?

File: README.md [hunk 2]
```diff
@@ -740,6 +740,18 @@ the mission but there can be a delay of up to 30 seconds after activation for th
 
 You can also change the **name of a unit*** (unit, not group) to include "**hpriority**" to make it high priority for the JTAC, or "**priority**" to set it to be medium priority. JTAC's will prioritize targets within view by first marking hpriority targets, then priority targets, and finally all others. This works seemlessly with the all/vehicle/troop functionality as well. In this way you can have them lase SAMS, then AAA, then armor, or any other order you decide is preferable.
 
+If the `DCS-SimpleTextToSpeech.lua` script is loaded, and configures (i.e. the `STTS.DIRECTORY`, `STTS.SRS_PORT` and optionaly the `STTS.GOOGLE_CREDENTIALS` variables are set), the JTAC can talk over SRS.
```
Comment (ciribob) [inline README.md:743]: Please add a link to https://github.com/ciribob/DCS-SimpleTextToSpeech

File: CTLD.lua [hunk 3]
```diff
@@ -5139,9 +5139,32 @@ ctld.jtacCurrentTargets = {}
...
-function ctld.JTACAutoLase(_jtacGroupName, _laserCode, _smoke, _lock, _colour)
+    local _radio = _radio
+    if not _radio then
+        _radio = {}
+        if _laserCode then
+            local _laserCode = tonumber(_laserCode)
+            if _laserCode and _laserCode >= 1111 and _laserCode <= 1688 then
+                local _laserB = math.floor((_laserCode - 1000)/100)
+                local _laserCD = _laserCode - 1000 - _laserB*100
+                local _frequency = tostring(30+_laserB+_laserCD*0.05)
```
Comment (ciribob) [inline CTLD.lua:5155]: This is really clever! 👍

**Comment by davidp57** [comment]
All done !

File: CTLD.lua (hunk 3 above)
Comment (davidp57) [reply CTLD.lua:5155]: Thanks 😊

**Comment by davidp57** [comment]
All done !

**Comment by ciribob** [comment]
Merged! Thank you !

**Pull request merged**